targets/TARGET_STM/sleep.c

```diff
@@ -84,6 +85,10 @@ void hal_deepsleep(void)
     // After wake-up from STOP reconfigure the PLL
     SetSysClock();
 
+    TIM_HandleTypeDef TimMasterHandle;
+    TimMasterHandle.Instance = TIM_MST;
+    __HAL_TIM_SET_COUNTER(&TimMasterHandle, EnterTimeUS);
```

Comment:
Does this mean that ticker is restarted and we resume the ticker from the last known timestamp?